Makefile

```diff
@@ -1,96 +1,79 @@
-OUTPUT := .output
-TARGET := event_monitor_ebpf
+.PHONY: all
+all: build
+
+# environment:
+ARCH := $(shell uname -m)
```

Comment:
Maybe we better use conditional assignment here:
ARCH ?= 
WDYT?
Same for tools like LLC, CLANG and LLVM-STRIP
This way, the user will be able to change the behavior from cmdline without editing the makefile

Reply:
the user can change override vars regardless of how they're assigned. the ?= is just if we want to leave it undefined (which we aren't in this case)

Reply:
Are you sure this will not override what the user chosen?
From what I understand, ?= is conditionaly setting the variable if wasn't set by the user, but I may be wrong here:
https://stackoverflow.com/questions/53369903/use-environment-variable-if-set-otherwise-use-default-value-in-makefile

Reply:
I just tested and it works.. also written here:
> When you override a variable with a command line argument, you can define either a recursively-expanded variable or a simply-expanded variable
https://www.gnu.org/software/make/manual/html_node/Overriding.html#Overriding

Reply:
having said that, there might be a different if people want to use environment variable to override cars.. I'll check

Reply:
`make build LLC=myllc` works with `:=` but `LLC=myllc make build` doesn't. I'll change